app/models/configuration.rb

```diff
@@ -41,28 +41,35 @@ def self.signups_enabled?
 
 
   # --------------------------------------------------------------- admin:paths
+  # In CE ./templates/ is always a folder (created by bin/setup) but in Pro
+  # it can be a symlink (if we're in Production). We use .realdirpath to
+  # ensure we're using the "shared" folder (that's the target of the link).
+  def self.paths_templates
+    @@paths_templates ||= Rails.root.join('templates').realdirpath
+  end
+
   def self.paths_templates_methodologies
-    create_with(value: Rails.root.join('../../shared/templates/methodologies/').to_s)
+    create_with(value: paths_templates.join('methodologies').to_s)
       .find_or_create_by(name: 'admin:paths:templates:methodologies').value
   end
 
   def self.paths_templates_notes
-    create_with(value: Rails.root.join('../../shared/templates/notes/').to_s)
+    create_with(value: paths_templates.join('notes').to_s)
       .find_or_create_by(name: 'admin:paths:templates:notes').value
   end
 
   def self.paths_templates_plugins
-    create_with(value: Rails.root.join('templates', 'plugins').to_s)
+    create_with(value: paths_templates.join('plugins').to_s)
       .find_or_create_by(name: 'admin:paths:templates:plugins').value
   end
 
   def self.paths_templates_projects
-    create_with(value: Rails.root.join('../../shared/templates/projects/').to_s)
+    create_with(value: paths_templates.join('projects').to_s)
       .find_or_create_by(name: 'admin:paths:templates:projects').value
   end
 
   def self.paths_templates_reports
-    create_with(value: Rails.root.join('templates', 'reports'))
+    create_with(value: paths_templates.join('reports'))
```

Comment:
This is missing the `to_s` call but it doesn't seem to be needed when I test it in the console. I suggest we skip it entirely but this needs to be consistent with the rest.

Reply:
Addressed, you could have saved me a commit w/ a suggestion ;-)